packages/loaders/stories/2-Dots.stories.tsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { Story, Meta } from '@storybook/react';
+import { PALETTE } from '@zendeskgarden/react-theming';
+import { Dots, IDotsProps } from '@zendeskgarden/react-loaders';
+
+export default {
+  title: 'Components/Loaders/Dots',
+  component: Dots
+} as Meta;
+
+const StyledExampleWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
```

Comment:
Isn't this just a Garden `<Row>` with `justifyContent="center"`?

Reply:
I've moved this to a Grid implementation